### ISSUE TITLE
License clarification for contributing

### ISSUE THREAD

**Issue by stijnherreman** [akkadotnet/akka.net#4137]
I'm looking for some clarifications regarding contributing:

* https://github.com/akkadotnet/akka.net/blob/dev/README.md says Akka.NET is part of the .NET Foundation, but the project is not listed on https://dotnetfoundation.org/projects
* Contributing to .NET Foundation projects (afaik) requires signing a CLA. There is https://github.com/akkadotnet/akka.net/issues/791 but I can't find the follow-up for that issue. PRs don't seem to have a bot for CLA signing.
* Source files list the .NET Foundation, e.g. `Copyright (C) 2013-2019 .NET Foundation <https://github.com/akkadotnet/akka.net>`
* https://github.com/akkadotnet/akka.net/blob/dev/CONTRIBUTING.md does not mention .NET Foundation

Is the transition to a .NET Foundation project an ongoing effort? Do contributors need to sign a CLA?

**Comment by Aaronontheweb** [comment]
1 Transition to the .NET Foundation is definitely complete (see https://github.com/akkadotnet/akka.net/pull/3189, from @jongalloway who was the DNF director at the time), but the DNF hasn't added us to their website. We've asked about that several times. I don't know what the hold-up is there.

2 As part of joining the DNF, we updated our copyright headers to list the DNF organization as the copyright holder. 

3 We haven't added the CLA-bot yet because I believe we need to add some DNF access to our organization - I dropped the ball on setting that up in 2019. That being said, a CLA isn't as necessary for [Apache 2.0 projects due to the language in section 5](http://www.apache.org/licenses/LICENSE-2.0):

> 5. Submission of Contributions. Unless You explicitly state otherwise, any Contribution intentionally submitted for inclusion in the Work by You to the Licensor shall be under the terms and conditions of this License, without any additional terms or conditions. Notwithstanding the above, nothing herein shall supersede or modify the terms of any separate license agreement you may have executed with Licensor regarding such Contributions.

A CLA more or less says the same thing. That being said, it's still a good idea to have it.

4. We should update the `CONTRIBUTING.md` file to reflect that - good point. I don't think we've touched that in years (i.e. most of the stuff about managing Git branches is handled automatically now by Github.)

TL;DR;
---
So long story short, you don't need to sign a .NET Foundation CLA to contribute to Akka.NET - the Apache 2.0 license automatically takes care of that intellectual property issues the CLA is designed to resolve, but we should / will add the CLA-BOT to our repositories as standard practice moving forward.

**Comment by Aaronontheweb** [comment]
I'll link some issues from the DNF to here to address some of these, so I'll reopen the issue until those get resolved.

**Comment by Aaronontheweb** [comment]
Should be able to resolve item 1 by submitting a PR here: https://github.com/dotnet/foundation/tree/master/projects

**Comment by nagytech** [comment]
> Should be able to resolve item 1 by submitting a PR here: https://github.com/dotnet/foundation/tree/master/projects

https://github.com/dotnet/foundation/issues/159

**Comment by Aaronontheweb** [comment]
Thanks @nagytech !

**Comment by Aaronontheweb** [comment]
This got resolved earlier this year with the DNF website, so I feel like we can close this now.
